DeepCat.js

```diff
@@ -473,14 +473,9 @@
 		return false;
 	}
 
-	function enableImeAndSuggestions() {
-		$( '.suggestions' ).css( 'z-index', 'auto' );
-		$( '.imeselector' ).css( 'z-index', 'auto' );
-	}
-
-	function disableImeAndSuggestions() {
-		$( '.suggestions' ).css( 'z-index', -1 );
-		$( '.imeselector' ).css( 'z-index', -1 );
+	function setImeAndSuggestions( enable ) {
```

Comment:
if we decide to do so (evil bool param!) I'd suggest to use verb "toggle" instead of "set" in the method name.